src/modules/settings/ThemePage.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Container } from '@mui/material';
+import { HalfTypeNavigation } from '../../shared-components/Navigation/TopNavigation/HalfTypeNavigation';
+import { colors } from 'src/styles/colors';
+import { ListItemSelect } from '../../shared-components/Card/ListItems/ListItemSelect';
+import { useState } from 'react';
+import { UserActionType, useUserDispatch } from 'src/context/UserContext';
+import { Theme } from 'src/types/Theme';
+
+export const ThemePage = () => {
+  const [selectedTheme, setSelectedTheme] = useState<Theme | null>(null);
```

Comment:
Same here, use context instead of local state.

Reply:
fixed